daylay92/entrack-backend
# Add basic app configuration

#### What does this PR do?
Add most of the essential configurations for the Application.
#### Description of Task to be completed?
- Setup unit testing.
- Add code linting with Airbnb style guide and ESlint.
- Setup redis database for testing, development and production environments.
- Add hound CI for code quality review of PR.
- Write unit test.
#### How should this be manually tested?
- Clone the repo and cd into the project folder
- Checkout to `ch-setup` branch and run `npm install`.
- Run `npm test` to see results. Also, inspect the file changes.
